sdk/js-cloud-server/src/types.ts

```diff
@@ -97,10 +97,10 @@ export type { DVCJSON, DVCCustomDataJSON }
 
 export type DVCVariableSet = Record<
     string,
-    Omit<DVCVariable, 'defaultValue' | 'isDefaulted'> & { _id: string }
+    Omit<DVCVariableInterface, 'defaultValue' | 'isDefaulted'> & { _id: string }
 >
 
-export interface DVCVariable {
+export interface DVCVariableInterface {
```

Comment:
Renaming this to `DVCVariableInterface` as that's what its renamed to in the one place its used.